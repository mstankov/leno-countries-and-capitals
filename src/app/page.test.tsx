import { render } from "@testing-library/react";
import Home from "./page";

describe("Home", () => {
  it("should match snapshot", () => {
    const { container } = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});
