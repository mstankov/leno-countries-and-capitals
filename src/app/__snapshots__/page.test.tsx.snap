// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match snapshot 1`] = `
<div>
  <main
    class="MuiBox-root css-0"
  >
    <div
      class="MuiBox-root css-xi606m"
    >
      <h2
        class="MuiTypography-root MuiTypography-h2 css-co5uzt-MuiTypography-root"
        data-testid="home-page-title"
      >
        Countries and Capitals
      </h2>
      <div
        class="MuiBox-root css-1821gv5"
      >
        <div
          class="MuiBox-root css-1lk7evk"
        >
          <h4
            class="MuiTypography-root MuiTypography-h4 css-pxw7rc-MuiTypography-root"
          >
            Rules
          </h4>
          <h6
            class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
          >
            Match each country with its respective capital
          </h6>
          <h6
            class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
          >
            The game lasts until all and capitals are matched
          </h6>
          <div
            class="MuiBox-root css-r9sjpm"
          >
            <span
              class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
            >
              A correct guess adds
               
              <span
                class="MuiTypography-root MuiTypography-caption css-1yjifjv-MuiTypography-root"
              >
                7
              </span>
               
              points
            </span>
            <span
              class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
            >
              An incorrect guess subtracts
               
              <span
                class="MuiTypography-root MuiTypography-caption css-1yjifjv-MuiTypography-root"
              >
                5
              </span>
               
              points
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root css-1h0qwx1"
    >
      <a
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-1yaqcqa-MuiButtonBase-root-MuiButton-root"
        data-testid="btn-start-new-game"
        href="/game"
        tabindex="0"
      >
        Start
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </a>
    </div>
  </main>
</div>
`;
